eppo_client/rules.py

```diff
@@ -19,30 +19,26 @@ class OperatorType(Enum):
 
 class Condition(SdkBaseModel):
     operator: OperatorType
-    attribute: str
+    attribute: Any
```

Comment:
can a type be set?

Reply:
Cleaned this up -- a bit, mypy isn't quite smart enough